addons/html_editor/static/src/core/history_plugin.js

```diff
@@ -1027,4 +1031,32 @@ export class HistoryPlugin extends Plugin {
         _map.set(node, serializedNode.nodeId);
         return [node, _map];
     }
+
+    _onDocumentKeydown(ev) {
+        const el = ev.target;
+        if ([el, ...ancestors(el)].some((element) => element === this.editable)) {
+            return;
+        }
+        if (isKeyboardEventUndo(ev) || isKeyboardEventRedo(ev)) {
+            this._onKeyupResetContenteditableNodes.push(
+                ...this.editable.querySelectorAll("[contenteditable=true]")
+            );
+            if (this.editable.getAttribute("contenteditable") === "true") {
+                this._onKeyupResetContenteditableNodes.push(this.editable);
+            }
+
+            for (const node of this._onKeyupResetContenteditableNodes) {
+                node.setAttribute("contenteditable", false);
+            }
+        }
+    }
+
+    _onDocumentKeyup() {
+        if (this._onKeyupResetContenteditableNodes.length) {
+            for (const node of this._onKeyupResetContenteditableNodes) {
+                node.setAttribute("contenteditable", true);
+            }
+            this._onKeyupResetContenteditableNodes = [];
+        }
+    }
```

Comment:
Do you add some mutation or it s filter ?

Reply:
Indeed, it's filtered